src/Helpers/FiltersHelper.php

```diff
@@ -142,7 +142,7 @@ private static function processAdditionalFilters($request)
         }
 
         // Process 'title' filter
-        if ($request->has('title')) {
+        if ($request->has('title') && $request->get('title') !== null) {
```

Comment:
I _think_ this could also be handled in one by using [filled](https://laravel.com/docs/9.x/requests#determining-if-input-is-present)
```suggestion
        if ($request->filled('title')) {
```